### ISSUE TITLE
#891604022------------- Links must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1756]
<strong>PageName:</strong> BigDataPage  

<strong>Source:</strong>`<a href="/bigdata/8ac198c3717a47dcbce461a2c8aa9f00/">`

<strong>Element Location: </strong>.sc-ePDLzJ.iBkgkw:nth-child(1) > td:nth-child(5) > .sc-gdyeKB.fywSFR > a

<strong>Impact: </strong>serious

<strong>Failure Summary:</strong> Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

<strong>Tags:</strong> cat.name-role-value,wcag2a,wcag412,wcag244,section508,section508.22.a,ACT

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/link-name?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\BigDataPage\891604022_Links must have discernible text.png`
